app/src/main/java/org/oppia/app/administratorcontrols/appversion/AppVersionActivityPresenter.kt

```diff
@@ -0,0 +1,34 @@
+package org.oppia.app.administratorcontrols.appversion
+
+import androidx.appcompat.app.AppCompatActivity
+import androidx.appcompat.widget.Toolbar
+import org.oppia.app.R
+import org.oppia.app.activity.ActivityScope
+import javax.inject.Inject
+
+/** The presenter for [AppVersionActivity]. */
+@ActivityScope
+class AppVersionActivityPresenter @Inject constructor(private val activity: AppCompatActivity) {
+  fun handleOnCreate() {
+    activity.setContentView(R.layout.app_version_activity)
+    setToolbar()
+    if (getAppVersionFragment() == null) {
+      activity.supportFragmentManager.beginTransaction().add(
+        R.id.app_version_fragment_placeholder,
+        AppVersionFragment()
+      ).commitNow()
+    }
+  }
+
+  private fun setToolbar() {
```

Comment:
This is correct but I think the title as well as the back button can be managed in the XML file itself, check the implementation in other files to correctly do that.

Reply:
Done